UI/signup.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Maintenance tracker | Signup</title>
+  <link rel="stylesheet" href="vendors/css/normalize.css" type="text/css">
+  <link rel="stylesheet" href="vendors/css/grid.css" type="text/css">
+  <link href="https://fonts.googleapis.com/css?family=Lato:100,300,300i,400" rel="stylesheet" type="text/css">
+  <link rel="stylesheet" href="https://cdn.materialdesignicons.com/2.3.54/css/materialdesignicons.min.css" type="text/css">
+  <link rel="stylesheet" href="css/style.css" type="text/css">
+</head>
+<body>
+  <div class="content">
+    <header>
+      <nav class="container">
+        <div class="row">
+          <div class="header-logo">
+            <img src="img/logo-white.png" alt="maintanace-tracker logo" class="logo">
+            <button type="button" id="navbar-toggler" class="nav-toggle"><i class="mdi mdi-view-sequential"></i></button>
+          </div>
+          <ul class="main-nav" id="navbar-collapse">
+            <li><a href="signup.html" class="active">SignUp</a></li>
+            <li><a href="signin.html">Login</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <section class="section-form">
+      <div class="row">
+        <form method="post" class="signup-form">
+          <h2 class="form-title">Create an account</h2>
+          <div class="row">
+            <div class="col span-1-of-3 label">
+                <label for="email">Email</label>
+            </div>
+            <div class="col span-2-of-3">
+                <input type="email" name="email" id="email" placeholder="Your email address" required>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col span-1-of-3 label">
+                <label for="password">Password</label>
+            </div>
+            <div class="col span-2-of-3">
+                <input type="password" name="password" id="password" placeholder="Your password" required>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col span-1-of-3 label">
+                <label for="confirm-password">Confirm Password</label>
+            </div>
+            <div class="col span-2-of-3">
+                <input type="password" name="confirm-password" id="confirm-password" placeholder="Confirm your password" required>
+            </div>
+          </div>
+
+          <div class="row">
```

Comment:
You should align your button to the center instead of the left. This abides by the web accessibility matrix

Reply:
Okay, I'll fix that